src/main/index.js

```diff
@@ -597,6 +605,121 @@ function runApp() {
     }
   })
 
+  function manageTray(window, removeWindow = false) {
+    function click(window, close = false) {
```

Comment:
By the looks of it the `click` and `createContextMenu` functions don't actually have to be inside of the `manageTray` function, so it would be better to move them into the parent scope so that they don't get recreated for every call to `manageTray`. Also stating the obvious, please pick suitable names that indicate they are related to the tray when you move them.